packages/client/src/_types/global/GlobalTypes.ts

```diff
@@ -1,9 +1,8 @@
+import type { GameManager } from "@backend/GameLogic/GameManager";
+import type { GameUIManager } from "@backend/GameLogic/GameUIManager";
 import type { Rectangle } from "@df/types";
 import type { Dispatch, SetStateAction } from "react";
 
-import type GameManager from "../../Backend/GameLogic/GameManager";
-import type GameUIManager from "../../Backend/GameLogic/GameUIManager";
```

Comment:
Here we are trying to import defaults in ESM one usually don't export defaults 🤷